Modix.Data/Models/Core/AuthorizationClaim.cs

```diff
@@ -148,5 +148,10 @@ public enum AuthorizationClaim
         /// </summary>
         [ClaimInfo(TagActions, "Authorizes a request to maintain a tag that was not created by the requesting user.")]
         MaintainOtherUserTag,
+        /// <summary>
+        /// Authorizes a request to create a giveaway and determine its winners.
+        /// </summary>
+        [ClaimInfo(Misc, "Authorizes a request to create a giveaway and determine its winners.")]
+        HostGiveaway,
```

Comment:
Nit: The giveaway command only looks at an existing message and randomizes the reactors to generate winners, and make an announcement, yes? `ExecuteGiveaway` or something similar seems more appropriate.

Reply:
Works for me